changelog.txt

```diff
@@ -6,6 +6,7 @@
 - Add: Feature toggle to disable Analytics UI #7168
 - Add: Hook reference slotFill support #6833
 - Add: Adding tests for PaymentGatewaySuggestions > List component #7201
+- Add: Show Pinterest in installed marketing extensions (if installed) #7417
```

Comment:
@haszari Just an fyi we use the Jetpack changelog entry cli now for changelog entries. You can run `npm run changelogger -- add` and follow the cli prompts to create a changelog entry file.